tests/mail/test_sendtestemail.py

```diff
@@ -2,32 +2,83 @@
 
 from django.core import mail
 from django.core.management import call_command
-from django.test import SimpleTestCase
+from django.test import SimpleTestCase, override_settings
 
 
+@override_settings(
+    ADMINS=(('Admin', 'admin@example.com'), ('Admin and Manager (Admin)', 'admin_and_manager@example.com')),
+    MANAGERS=(('Manager', 'manager@example.com'), ('Admin and Manager (Manager)', 'admin_and_manager@example.com')),
```

Comment:
would be nice to make this line a bit shorter so it doesn't run past 119 chars. the names aren't so important now that they aren't included in the mail